### PR TITLE
WikiaApiQueryDomains: do not allow requests with wkfrom > wkto

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-2022

Return this in such cases:

```json
{
	servedby: "dev-kopytko",
	error: {
		code: "wk",
		info: "wkto should be equal or greater than wkfrom"
	}
}
```

instead of DB error exception (negative value was passed to `LIMIT`).

### CODE REVIEW

File: extensions/wikia/WikiaApi/WikiaApiQueryDomains.php [hunk 1]
```diff
@@ -29,6 +29,7 @@ protected function getDB() {
 	public function execute() {
```
Comment (WikiaTech) [inline extensions/wikia/WikiaApi/WikiaApiQueryDomains.php:29]: MAJOR The Cyclomatic Complexity of this function "execute" is 22 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 11 issues
* ![MAJOR][MAJOR] 7 major
* ![MINOR][MINOR] 3 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L50](https://github.com/Wikia/app/blob/7d864a1fae1c4f5a05fc2d8cc307d57151d09503/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L50): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L51](https://github.com/Wikia/app/blob/7d864a1fae1c4f5a05fc2d8cc307d57151d09503/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L51): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L55](https://github.com/Wikia/app/blob/7d864a1fae1c4f5a05fc2d8cc307d57151d09503/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L55): Merge this if statement with the enclosing one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/7d864a1fae1c4f5a05fc2d8cc307d57151d09503/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): Merge this if statement with the enclosing one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/7d864a1fae1c4f5a05fc2d8cc307d57151d09503/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L87](https://github.com/Wikia/app/blob/7d864a1fae1c4f5a05fc2d8cc307d57151d09503/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L87): Take the required action to fix the issue indicated by this "FIXME" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L50](https://github.com/Wikia/app/blob/7d864a1fae1c4f5a05fc2d8cc307d57151d09503/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L50): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L51](https://github.com/Wikia/app/blob/7d864a1fae1c4f5a05fc2d8cc307d57151d09503/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L51): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/7d864a1fae1c4f5a05fc2d8cc307d57151d09503/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![INFO][INFO] [WikiaApiQueryDomains.php#L8](https://github.com/Wikia/app/blob/7d864a1fae1c4f5a05fc2d8cc307d57151d09503/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L8): Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by mszabo-wikia — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 11 issues
* ![MAJOR][MAJOR] 7 major
* ![MINOR][MINOR] 3 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L50](https://github.com/Wikia/app/blob/c00354e77348408602b3604b75d27d55296dd1f2/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L50): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L51](https://github.com/Wikia/app/blob/c00354e77348408602b3604b75d27d55296dd1f2/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L51): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L55](https://github.com/Wikia/app/blob/c00354e77348408602b3604b75d27d55296dd1f2/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L55): Merge this if statement with the enclosing one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/c00354e77348408602b3604b75d27d55296dd1f2/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): Merge this if statement with the enclosing one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/c00354e77348408602b3604b75d27d55296dd1f2/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L87](https://github.com/Wikia/app/blob/c00354e77348408602b3604b75d27d55296dd1f2/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L87): Take the required action to fix the issue indicated by this "FIXME" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L50](https://github.com/Wikia/app/blob/c00354e77348408602b3604b75d27d55296dd1f2/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L50): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L51](https://github.com/Wikia/app/blob/c00354e77348408602b3604b75d27d55296dd1f2/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L51): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/c00354e77348408602b3604b75d27d55296dd1f2/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![INFO][INFO] [WikiaApiQueryDomains.php#L8](https://github.com/Wikia/app/blob/c00354e77348408602b3604b75d27d55296dd1f2/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L8): Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 11 issues
* ![MAJOR][MAJOR] 7 major
* ![MINOR][MINOR] 3 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L50](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L50): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L51](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L51): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L55](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L55): Merge this if statement with the enclosing one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): Merge this if statement with the enclosing one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L87](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L87): Take the required action to fix the issue indicated by this "FIXME" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L50](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L50): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L51](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L51): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![INFO][INFO] [WikiaApiQueryDomains.php#L8](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L8): Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 11 issues
* ![MAJOR][MAJOR] 7 major
* ![MINOR][MINOR] 3 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L50](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L50): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L51](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L51): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L55](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L55): Merge this if statement with the enclosing one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): Merge this if statement with the enclosing one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [WikiaApiQueryDomains.php#L87](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L87): Take the required action to fix the issue indicated by this "FIXME" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L50](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L50): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L51](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L51): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![MINOR][MINOR] [WikiaApiQueryDomains.php#L61](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L61): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. ![INFO][INFO] [WikiaApiQueryDomains.php#L8](https://github.com/Wikia/app/blob/18a6104ce9348bccd9a224d5f5d12b716b29feef/extensions/wikia/WikiaApi/WikiaApiQueryDomains.php#L8): Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**